spyder/plugins/findinfiles/plugin.py

```diff
@@ -53,24 +53,30 @@ def __init__(self, parent=None):
         case_sensitive = self.get_option('case_sensitive')
         path_history = self.get_option('path_history', [])
         search_in_index = self.get_option('search_in_index', default=0)
+        max_results = self.get_option('max_results', default=1000)

```

Comment:
Please add this option to our DEFAULTS dict in `config/main.py` and remove the `default=1000` from here.

Reply:
Ok